.pre-commit-config.yaml

```diff
@@ -1,15 +1,15 @@
 repos:
 - repo: https://github.com/asottile/pyupgrade
-  rev: v2.7.2
+  rev: v2.7.3
   hooks:
   - id: pyupgrade
-    args: ["--py3-plus", "--keep-mock"]
+    args: ["--py36-plus", "--keep-mock"]
```

Comment:
Do we still need `--keep-mock`?

Reply:
Good catch! I don't think it does anything now that we're not using the backport. I waited to reply until I got to confirm this.
I just opened #556 to clean up.